source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -59,6 +59,13 @@
 #define CHECK(condition)                                                                           \
   QUIC_LOG_IF_IMPL(FATAL, ABSL_PREDICT_FALSE(!(condition))) << "CHECK failed: " #condition "."
 
+#define CHECK_GT(a, b) CHECK((a) > (b))
+#define CHECK_GE(a, b) CHECK((a) >= (b))
+#define CHECK_LT(a, b) CHECK((a) < (b))
+#define CHECK_LE(a, b) CHECK((a) <= (b))
+#define CHECK_NE(a, b) CHECK((a) != (b))
+#define CHECK_EQ(a, b) CHECK((a) == (b))
+
```

Comment:
This is also added in #6650

Reply:
Yeah, I copied it from there:)  I copied it to the exact same location so it shouldn't cause merge conflicts.